frappe/modules/import_file.py

```diff
@@ -107,6 +107,10 @@ def import_doc(docdict, force=False, data_import=False, pre_process=None,
 
 	doc = frappe.get_doc(docdict)
 
+	if hasattr(doc.meta, 'is_tree') and getattr(doc.meta, 'is_tree'):
```

Comment:
I feel we should show some sort of message saying hey we're ignoring `lft` and `rgt` fields for these docs. What do you think?

---

Also, another way to do this

```suggestion
	if getattr(doc.meta, 'is_tree', None):
```

Reply:
Is there any scenario where you would want to import the values of lft and rgt? As far as I can tell they are only used internally to maintain the tree structure.

Reply:
Good question. I think there should be no implicit actions or "magic" in any process; Even when I export a tree to debug issues in its broken tree structure to test it out on a local instance. I should be informed, _(maybe just via the CLI or/and in development mode?)_ or maybe this doesn't matter at all.

But, since this is in the context of importing fixtures, shouldn't we *not* export internal fields like `lft`, `rgt` to begin with? This seems to be an issue related to the export of fixtures rather than an import problem to me.

Reply:
Fair enough. I'll remove them from the export as well, print a message to the console, comment the code and add it to the docs.